src/hyp3_isce2/insar_tops_burst.py

```diff
@@ -126,6 +118,64 @@ def make_parameter_file(
         json.dump(output, f)
 
 
+def translate_outputs(product_dir: Path, product_name: str):
+    """Translate ISCE outputs to a standard GTiff format with a UTM projection
+
+    Args:
+        product_dir: Path to the ISCE merge directory
+        product_name: Name of the product
+    """
+    ISCE2_DATASET = namedtuple('ISCE2_DATASET', ['name', 'suffix', 'band'])
+    datasets = [
+        ISCE2_DATASET('filt_topophase.unw.geo', 'unw_phase', 2),
+        ISCE2_DATASET('phsig.cor.geo', 'corr', 1),
+        ISCE2_DATASET('z.rdr.full.geo', 'dem', 1),
+        ISCE2_DATASET('filt_topophase.unw.conncomp.geo', 'conncomp', 1),
+        ISCE2_DATASET('filt_topophase.flat.geo', 'wrapped_phase', 1),
+    ]
+
+    for dataset in datasets:
```

Comment:
I think it might read a bit cleaner if we handled the `wrapped_phase` dataset outside of this loop (either before or after the loop), and removed it from the `datasets` list, so that this loop *only* performs the `gdal.Translate` calls.

Reply:
Yeah, I went back and forth on this. With your vote, I'll refactor to remove the `if` statement